pkg/storage/segments/store.go

```diff
@@ -217,6 +218,31 @@ func (s *segmentStore) Get(ctx context.Context, path storj.Path) (
 			return nil, Meta{}, err
 		}
 
+		// get the total non nil nodes
+		totalNonNilNodes := 0
+		for _, v := range nodes {
+			if v != nil {
+				totalNonNilNodes++
+			}
+		}
+
+		// calculate how many minimum nodes needed based on t = k + (n-o)k/o
+		t := es.RequiredCount() + ((es.TotalCount()-totalNonNilNodes)*es.RequiredCount())/totalNonNilNodes
```

Comment:
Shouldn't the minimum nodes needed still be the same. As far as I understand the issue, it should be based on the parameters not the results of lookup.